### PR TITLE
sc-86: Sc 86/profile tests

### PR DESCRIPTION
**Pull request by Justin1002**
Added unit testing for the following components related to Profile:

Profile.jsx
AccountInfo.jsx
NoItems.jsx
ProfileBanner.jsx
ProfileCarousel.jsx

### CODE REVIEW

**Review by akshay-rakheja — DISMISSED** [review]

File: frontend/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```
Comment (connie-ho) [inline frontend/src/index.jsx:1]: oops I think this might be an extra file

File: frontend/src/components/profile/__tests__/NFTList.test.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import seedrandom from 'seedrandom';
+
+import NFTList from '../NFTList';
+
+const seed = new seedrandom(`BEGINNERBLOCS${Math.random()*100}`);
+
+const mockProps = {
+  items: [
+    {
+      address:randomHexString(seed, 0, 32),
```
Comment (connie-ho) [inline frontend/src/components/profile/__tests__/NFTList.test.jsx:13]: you can add the extra fields to the createTestNft fucntion to be consisitent

File: frontend/src/components/profile/__tests__/ProfileBanner.test.jsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+import { renderWithProviders } from '../../../lib/test-utils'
+import { randomHexString } from '@ethersproject/testcases'
+import seedrandom from 'seedrandom';
+
+const seed = new seedrandom(`BEGINNERBLOCS${Math.random()*100}`);
+
+const mockProps = {
+  nfts: [
+    {
+      address:randomHexString(seed, 0, 32),
```
Comment (connie-ho) [inline frontend/src/components/profile/__tests__/ProfileBanner.test.jsx:11]: same ting here

**Review by connie-ho — APPROVED** [review]
tanks <3333

**Review by connie-ho — APPROVED** [review]

**Pull request merged**